src/main/java/org/gradle/profiler/studio/launcher/StudioConfigurationProvider.java

```diff
@@ -31,8 +31,12 @@ public class StudioConfigurationProvider {
     private static final List<String> DEFAULT_MACOS_JAVA_PATHS = ImmutableList.of("Contents/jre/Contents/Home/bin/java", "Contents/jbr/Contents/Home/bin/java");
     private static final List<String> DEFAULT_WINDOWS_JAVA_PATHS = ImmutableList.of("jre/bin/java.exe", "jbr/bin/java.exe");
     private static final List<String> DEFAULT_LINUX_JAVA_PATHS = ImmutableList.of("jre/bin/java", "jbr/bin/java");
-    private static final Pattern LINUX_CLASSPATH_LIB_PATTERN = Pattern.compile(".*CLASS_PATH=.*(?<lib>lib/.+\\.jar).*");
-    private static final Pattern WINDOWS_CLASSPATH_LIB_PATTERN = Pattern.compile(".*CLASS_PATH=.*(?<lib>lib\\\\.+\\.jar).*");
+
+    /**
+     * Android Studio Dolphin (2021.3) and older versions use "CLASSPATH", while Electric Eel (2022.1) and newer versions use CLASS_PATH
+     */
+    private static final Pattern LINUX_CLASSPATH_LIB_PATTERN = Pattern.compile(".*(CLASS_PATH|CLASSPATH)=.*(?<lib>lib/.+\\.jar).*");
+    private static final Pattern WINDOWS_CLASSPATH_LIB_PATTERN = Pattern.compile(".*(CLASS_PATH|CLASSPATH)=.*(?<lib>lib\\\\.+\\.jar).*");
```

Comment:
Nit: instead of `(CLASS_PATH|CLASSPATH)` you could use `(?:CLASS_PATH|CLASSPATH)` to make it obvious that this group is not captured.

Reply:
Fixed